src/optimisation/Optimisation.jl

```diff
@@ -228,11 +229,47 @@ end
 
 # Various StatsBase methods for ModeResult
 
-function StatsBase.coeftable(m::ModeResult; level::Real=0.95)
+function StatsBase.coeftable(m::ModeResult; level::Real=0.95, numerrors_warnonly::Bool=true)
```

Comment:
Since we introduce a new argument compared to the usual `StatsBase.coeftable` method, could this have a docstring?